hnn_core/network.py

```diff
@@ -225,6 +229,24 @@ def copy(self):
         net_copy._reset_drives()
         return net_copy
 
+    def _set_cell_properties(self):
+        '''Set cell properties by cell type
+
+        This relies on self.cellname_list and must be updated when introducting
+        new cell types to HNN.
+        '''
+        for cell_type in self.cellname_list:
+            if cell_type in ('L2_pyramidal', 'L5_pyramidal'):
+                p_secs, p_syn, topology, sect_loc = pyramidal(
```

Comment:
`cells_default.pyramidal` and `cells_default.basket` are only called from within `Network` now.